notebooks/community/ml_ops/stage3/get_started_with_dataflow_pipeline_components.ipynb

```diff
@@ -40,9 +40,17 @@
         "  </td>\n",
```

Comment:
<p>remove this bullet list:</p><p> </p><ul><li>Dataset</li><li>Pipeline</li><li>Model</li><li>Endpoint</li><li>AutoML Training Job</li><li>Batch Job</li><li>Custom Job</li><li>Hyperparameter Tuning Job</li><li>Cloud Storage Bucket</li></ul><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
@andrewferlitsch : Correct the list of cleanup items.